[2024-05-05 13:49:39,846] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_prostgres_hooks_v2.poastgres_to_s3 2024-04-14T00:00:00+00:00 [queued]>
[2024-05-05 13:49:39,900] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_prostgres_hooks_v2.poastgres_to_s3 2024-04-14T00:00:00+00:00 [queued]>
[2024-05-05 13:49:39,901] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-05-05 13:49:39,902] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-05-05 13:49:39,903] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-05-05 13:49:39,949] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): poastgres_to_s3> on 2024-04-14T00:00:00+00:00
[2024-05-05 13:49:39,961] {standard_task_runner.py:52} INFO - Started process 7830 to run task
[2024-05-05 13:49:39,971] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_prostgres_hooks_v2', 'poastgres_to_s3', '2024-04-14T00:00:00+00:00', '--job-id', '114', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpg5_8sw3z', '--error-file', '/tmp/tmp7wa4wlns']
[2024-05-05 13:49:39,982] {standard_task_runner.py:77} INFO - Job 114: Subtask poastgres_to_s3
[2024-05-05 13:49:40,087] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_prostgres_hooks_v2.poastgres_to_s3 2024-04-14T00:00:00+00:00 [running]> on host d4a0c764ecae
[2024-05-05 13:49:40,191] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mark
AIRFLOW_CTX_DAG_ID=dag_with_prostgres_hooks_v2
AIRFLOW_CTX_TASK_ID=poastgres_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2024-04-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-14T00:00:00+00:00
[2024-05-05 13:49:40,211] {base.py:74} INFO - Using connection to: id: postgres_localhost. Host: host.docker.internal, Port: 5432, Schema: test, Login: airflow, Password: XXXXXXXX, extra: None
[2024-05-05 13:49:40,241] {dag_with_postgres_hooks.py:32} INFO - Saved orders data in text file: dags/get_orders_20240414.txt
[2024-05-05 13:49:40,245] {base_aws.py:362} INFO - Airflow Connection: aws_conn_id=minio_conn
[2024-05-05 13:49:40,260] {base_aws.py:164} INFO - Credentials retrieved from extra_config
[2024-05-05 13:49:40,265] {base_aws.py:81} INFO - Creating session with aws_access_key_id=ROOTUSER region_name=None
[2024-05-05 13:49:40,297] {base_aws.py:151} INFO - role_arn is None
[2024-05-05 13:49:40,517] {dag_with_postgres_hooks.py:41} INFO - Orders file /tmp/tmp7q8kbzsr20240414 has been pushed to S3!
[2024-05-05 13:49:40,523] {python.py:118} INFO - Done. Returned value was: None
[2024-05-05 13:49:40,554] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_prostgres_hooks_v2, task_id=poastgres_to_s3, execution_date=20240414T000000, start_date=20240505T134939, end_date=20240505T134940
[2024-05-05 13:49:40,665] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-05 13:49:40,718] {local_task_job.py:146} INFO - Task exited with return code 0
